tests/compat.rs

```diff
@@ -1,85 +1,62 @@
-use super::*;
+use rpm::*;
+use std::fs::File;
 use std::io::prelude::*;
 use std::io::BufReader;
 use std::process::Stdio;
-use std::str::FromStr;
 
-fn test_rpm_file_path() -> std::path::PathBuf {
-    let mut rpm_path = cargo_manifest_dir();
-    rpm_path.push("test_assets/389-ds-base-devel-1.3.8.4-15.el7.x86_64.rpm");
-    rpm_path
-}
-
-fn cargo_manifest_dir() -> std::path::PathBuf {
-    std::path::PathBuf::from(env!("CARGO_MANIFEST_DIR"))
-}
+mod common;
 
-fn cargo_out_dir() -> std::path::PathBuf {
-    cargo_manifest_dir().join("target")
-}
-
-#[cfg(feature = "signature-meta")]
 use signature::{self, Verifying};
 
-#[cfg(feature = "signature-pgp")]
 mod pgp {
     use super::*;
-    use futures::io::AsyncWriteExt;
     use signature::pgp::{Signer, Verifier};
-    use tokio_util::compat::TokioAsyncReadCompatExt;
+    use std::str::FromStr;
 
-    #[tokio::test]
     #[serial_test::serial]
-    async fn create_full_rpm_async() -> Result<(), Box<dyn std::error::Error>> {
+    fn create_full_rpm() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
We test async building here already, so we may as well simplify the feature matrix for these tests: https://github.com/rpm-rs/rpm/pull/123/files#diff-a0262fea108af44a98cc49e5eb72641963dd816513f2d0a22eb738fcfed55ebeR303

Later we can just compare the output of async to not-async building, and that will give us coverage of async automatically, presuming we keep async around.